docs/pages/database-access/guides/aws-discovery.mdx

```diff
@@ -0,0 +1,176 @@
+---
+title: AWS Database Auto-Discovery
+description: How to configure Teleport to discover AWS-hosted databases.
+---
+
+(!docs/pages/includes/database-access/discovery-guide-brief.mdx cloud="AWS" !)
+
+<Admonition type="tip">
+
+Discovery Service can run independently on a different machine from Database
+Services.
+
+For example, you can run an instance of the Discovery Service for discovering
+databases across all your AWS accounts, and an instance of the Database Service
+in each VPC for accessing database servers in private subnets.
+
+The same Discovery Service can also be used for Azure database discovery,
+server instance discovery and Kubernetes discovery.
+
+</Admonition>
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- An AWS account with permissions to create and attach IAM policies.
+- One or more database servers hosted by AWS database services.
+- A host to run the Teleport Discovery Service.
+
+## Step 1/4. Generate a join token
+
+The Discovery Service require a valid auth token to connect to the cluster.
```

Comment:
I think we should replace "auth token" with "join token" to be consistent with the heading